.github/workflows/ci.yml

```diff
@@ -0,0 +1,86 @@
+name: ci
```

Comment:
nitpick: personal choice is to name files with what they help with. While CI is decent enough, i still like something in the format of `ci-on-pr-clippy-rustfmt-build.yml`